assets/src/edit-story/components/inspector/prepublish/checklistTab.js

```diff
@@ -277,13 +279,13 @@ const ChecklistTab = (props) => {
           key={id}
           pageGroup={pageGroup}
         >
-          {message}
-          <HelperText>{help}</HelperText>
-          {onClick && (
-            <VisuallyHidden>
-              {__('Select offending element', 'web-stories')}
-            </VisuallyHidden>
+          {onClick ? (
+            <Underline title={accessibleText}>{message}</Underline>
+          ) : (
+            message
           )}
+          <HelperText>{help}</HelperText>
+          {onClick && <VisuallyHidden>{accessibleText}</VisuallyHidden>}
```

Comment:
Should we combine the components rendered from checking the existence of `onClick`?
```suggestion
          {onClick ? (
            <>
              <Underline title={accessibleText}>{message}</Underline>
              <VisuallyHidden>{accessibleText}</VisuallyHidden>
            </>
          ) : (
            message
          )}
          <HelperText>{help}</HelperText>
```

Reply:
depends on the order that we want it to read on the screen when users arrow through the group. in this order screen readers will read the title > accessible text > message rather than title > message > accessible text.